components/textbox/index.js

```diff
@@ -11,15 +11,15 @@ import {autocompleteChannels} from 'actions/channel_actions';
 
 import Textbox from './textbox.jsx';
 
-const makeMapStateToProps = () => {
+const mapStateToProps = (state, ownProps) => {
```

Comment:
This should be reverted. `makeMapStateToProps` is ran once for each instance of `<Textbox/>` that appears in the app, and each `Textbox` needs their own copy of `getProfilesInChannel` and `getProfilesNotInChannel` so that we can memoize them independently. Currently, those selectors are recreated each time `mapStateToProps` is ran, so there's no memoization happening.

Reply:
This file should have had no changes anyway - thanks for the catch